helpers/deproxy.py

```diff
@@ -626,6 +626,7 @@ def __init__(self, is_ssl=False, proto="http/1.1"):
         self.want_write = True  # TLS ClientHello is the first one
         self.server_hostname = None
         self.context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        self.context.keylog_filename = "secrets.bin"
```

Comment:
As for me `secrets.bin` has confusing extension. NSS key log format is not binary. It's plain text.

Also, current approach is not reliable, we can't simply appending to `secrets.bin`. File will grow and potentially can exhaust all free space on disk. Have we to dump secrets for all runs, maybe would be better to run it only if some argument has been passed to script?

On CI we need to have secrets for corresponding pcap. But on local runs we do not need to save each run separately. Maybe worth to make two args, one will be used on CI for saving secrets for each run. And another one for local runs, that just appends secrets to file?